### PR TITLE
card fixes

### PR DESCRIPTION
**Pull request by Shan-barcha**

### CODE REVIEW

**Comment by hunzai** [comment]
can you add a link to nfts in the header?

File: components/Card.tsx [hunk 1]
```diff
@@ -3,35 +3,44 @@ import Image from "next/image";
...
   return (
-    <div className="flex justify-center mt-5 container mx-auto">
-      <div className="rounded-lg shadow-xl bg-white w-[100%] sm:max-w-[100%] lg:max-w-[300px] ">
+    <div className="flex rounded-xl border bg-gray-50 justify-center w-full mt-5 flex-col lg:flex-row  lg:max-w-[450px] xl:max-w-[420px]">
+      <div className="rounded-lg shadow-xl bg-white w-[100%]  ">
         <a href="#!">
           <Image
-            src="/../public/assects/9.png"
+            // src="/../public/assects/9.png"
+            src={`/QmQqzMTavQgT4f4T5v6PWBp7XNKtoPmC9jvn12WPT3gkSE`}
```
Comment (hunzai) [inline components/Card.tsx]: change this to https://ipfs.io/ipfs/QmQqzMTavQgT4f4T5v6PWBp7XNKtoPmC9jvn12WPT3gkSE

File: pages/api/fakedb.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
```
Comment (hunzai) [inline pages/api/fakedb.json:1]: create multiple json nfts to test. 
Add an array with json schema e.g. 3 times

File: components/CardList.tsx [hunk 3]
```diff
@@ -1,14 +1,19 @@
 import React from "react";
 import Card from "./Card";
 
-const CardListItem = ({ assets }) => {
+const CardListItem = ({ fakeData }) => {
+  // console.log(assets);
```
Comment (hunzai) [inline components/CardList.tsx]: remove commented code

File: next.config.js [hunk 4]
```diff
@@ -29,4 +29,15 @@ module.exports = {
     };
     return config;
   },
+  // images: {
```
Comment (hunzai) [inline next.config.js]: You can remove the code. This is only needed when we have a remote image server.

**Review by hunzai — CHANGES_REQUESTED** [review]
@Shan-barcha rename nft page to listNfts

**Comment by hunzai** [comment]
image

- List NFT link is going to /nft

**Comment by hunzai** [comment]
when I connect to wallet I always see the same NFTs. The nfts from the wallet are not shown 

 
image

**Comment by Shan-barcha** [comment]
This is what I'm trying to say, sir. Sir, the image is static and derived from fake data.

**Pull request merged**